core/templates/dev/head/services/IssuesService.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to help identify and manage issues in exploration
```

Comment:
I think I'm a bit unclear on what the exact boundaries of this service are.

What rule determines whether a function/method goes in this service or not, and what is the motivation for introducing it in the first place?

(Otherwise, code LGTM. Thanks!)

Reply:
So my motivation is to consolidate the logic other pages and eventually the Issues Tab will use to determine issues. For example, since the Issues Tab will also have logic for showing unresolved unaddressed answers, I also plan on creating a "getOutstandingUnaddressedAnswers" function which can be shared by the UnresolvedAnswerOverviewDirective and Issues Tab.

This also helps clean up a lot of the messy code in UnresolvedAnswerOverviewDirective which I'm not too happy about. I believe keeping it here will make it a lot simpler, and will set good examples for future directives we add to the overview (should we plan to add any).

Reply:
I see, OK. Agree with the principle of keeping logical code in a service where it can be more easily tested. Thanks!